lib/Runtime/Runtime.h

```diff
@@ -496,7 +496,9 @@ enum tagDEBUG_EVENT_INFO_TYPE
 #include "Library/JavascriptError.h"
 #include "Library/JavascriptArray.h"
 
+#include "Library\AtomicsObject.h"
```

Comment:
good catch. thanks

---
In reply to: [77396220](https://github.com/Microsoft/ChakraCore/pull/1533#discussion_r77396220) [](ancestors = 77396220)